frontend/src/flashcards/Folder.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import FlashcardFront from "./FlashcardFront";
+import { Link } from "react-router-dom";
+
+const folderStyle = {
+  borderRadius: "30px",
+  position: "absolute",
+  width: "300px",
+  height: "350px",
+  backgroundColor: "white",
+  color: "black",
+  boxShadow: "rgba(0, 0, 0, 0.2) 0px 1px 8px",
+  fontFamily: "'Montserrat', sans-serif",
+  fontSize: "30px",
+  fontWeight: "600",
+  textAlign: "center",
+  display: "flex",
+  flexDirection: "column",
+  justifyContent: "center",
+  alignItems: "center",
+};
+const outerStyle = {
+  width: "300px",
+  height: "350px",
+  margin: "20px",
+};
```

Comment:
If the CSS got any longer than this then I would consider maybe creating a seperate flashcards.css file.

Reply:
alright!